agent/src/main/java/com/teamscale/jacoco/agent/commit_resolution/git_properties/IGitPropertiesLocator.java

```diff
@@ -1,10 +1,12 @@
 package com.teamscale.jacoco.agent.commit_resolution.git_properties;
 
+import org.conqat.lib.commons.collections.Pair;
+
 import java.io.File;
 
 /** Interface for the locator classes that search JAR files for git.properties files containing certain properties. */
 public interface IGitPropertiesLocator {
 
 	/** Searches the JAR file for the git.properties file containing certain properties. */
-	void searchJarFileForGitPropertiesAsync(File jarFile);
+	void searchJarFileForGitPropertiesAsync(Pair<File, Boolean> jarFile);
```

Comment:
The doc comment should mention what the pair represents. For a plain file this was quite obvious, but the boolean flag needs more context.